dowhy/datasets.py

```diff
@@ -92,6 +92,150 @@ def linear_dataset(
     stddev_outcome_noise=0.01,
     one_hot_encode=False,
 ):
+"""
+Generate a synthetic dataset with a known effect size, ready for DoWhy analysis
+
+This function generates a pandas dataFrame with num_samples records. The variables follow a naming convention were the first letter indicates its role in the causality graph and then a sequence number.
+
+Parameters
+---------------
+Beta : int or list/ndarray of lenght num_treatments of type int
```

Comment:
the docstring needs to follow a specific format to be rendered on the docs website. can you reformat the text?
we are using the reST standard sphinx format. 

:param params1: description
:returns: description

you can also see the CausalModel.py class for example of the format.